### DISCUSSION TITLE
How to reduce the logs from health checks

### DISCUSSION THREAD

**Discussion by IgordeOliveira** [serversideup/docker-php#405] [Q&A] [answered]
Hey everyone, is it possible to increase the time for the health check? I'm getting two logs for the same request. I believe it's from /index.php in Nginx and /up in Laravel. I know Docker has an option to do this, but I don't think it's the best option since I would need to overwrite the image defaults.

**Comment by jaydrogers** [comment, marked as answer]
Check out v3.3.0. It has a much better experience. I disabled the FPM logs by default, so your Docker logs are a lot cleaner https://github.com/serversideup/docker-php/releases/tag/v3.3.0

It will be available once this build is complete 🤓 https://github.com/serversideup/docker-php/actions/runs/10166037146

**Reply by IgordeOliveira** [reply]
TY Jay 🙏
